core/controllers/editor_test.py

```diff
@@ -872,6 +855,10 @@ def _mock_logging_function(msg, *args):
             observed_log_messages[0],
             'Could not find state: invalid_state_name')
 
+        self.assertEqual(
```

Comment:
You might as well replace these three assertions with a single self.assertEqual(observed_log_messages, ...)

Reply:
Done